### ISSUE TITLE
flutter_sentry is using deprecated Flutter API

### ISSUE THREAD

**Issue by ksdme** [futureware-tech/flutter_sentry#13]
My current Flutter version:
```
Flutter 1.12.13+hotfix.5 • channel unknown • unknown source
Framework • revision 27321ebbad (5 weeks ago) • 2019-12-10 18:15:01 -0800
Engine • revision 2994f7e1e6
Tools • Dart 2.7.0
```

Deprecation notice:
```
Note: /home/ksdme/Tools/flutter/.pub-cache/hosted/pub.dartlang.org/device_info-0.4.1+4/android/src/main/java/io/flutter/plugins/deviceinfo/DeviceInfoP
lugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Running Gradle task 'assembleDebug'...                                  
warning: the '-d' option with a directory destination is ignored because '-Xbuild-file' is specified
/home/ksdme/Tools/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_sentry-0.1.0/android/src/main/kotlin/org/dasfoo/flutter_sentry/FlutterSentryPlugi
n.kt:17:54: warning: 'getFlutterEngine(): FlutterEngine' is deprecated. Deprecated in Java
    val channel = MethodChannel(flutterPluginBinding.getFlutterEngine().getDartExecutor(), "flutter_sentry")       
```

**Comment by dotdoom** [comment]
Thanks for reporting it.

Looks like there are 2 issues here -- one with device_info (which is an upstream package) and the one with flutter_sentry (which we can and should fix).

Specifically this:

```
flutter_sentry/FlutterSentryPlugin.kt:17:54: warning: 'getFlutterEngine(): FlutterEngine' is deprecated. Deprecated in Java
```

**Comment by ksdme** [comment]
Thanks for responding! I opened an upstream issue for `device_info` package. You can track it at 
 https://github.com/dasfoo/flutter_sentry/issues/13

**Comment by dotdoom** [comment]
The version containing the fix, v0.2.1, has been published! Marking this resolved, and if you notice that it's not, please feel free to comment!
